integration/client/client.go

```diff
@@ -472,7 +472,7 @@ func (c *Client) rangeQueryURL(query string) string {
 }
 
 func (c *Client) LabelNames(ctx context.Context) ([]string, error) {
-	ctx, cancelFunc := context.WithTimeout(ctx, time.Second)
+	ctx, cancelFunc := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
I changed it for consistency (because I used 30s in other places on that same code). But the overall thinking here is: if we mess up with something, those calls hang forever (it happened to me). This timeout is just to not make our pipeline takes forever is a similar thing happen.